plugins/node/instrumentation-runtime-node/README.md

```diff
@@ -62,7 +62,7 @@ nodejs_performance_event_loop_utilization 0.010140079547955264
 
 | name | type | unit | default | description |
 |---|---|---|---|---|
-| [`eventLoopUtilizationMeasurementInterval`](./src/types.ts#L25) | `int` | millisecond | `5000` | The approximate number of milliseconds for which to calculate event loop utilization averages. A larger value will result in more accurate averages at the expense of less granular data. Should be set to below the scrape interval of your metrics collector to avoid duplicated data points. |
+| [`monitoringPrecision`](./src/types.ts#L25) | `int` | millisecond | `5000` | The approximate number of milliseconds for which to calculate event loop utilization averages. A larger value will result in more accurate averages at the expense of less granular data. Should be set to below the scrape interval of your metrics collector to avoid duplicated data points. |
```

Comment:
**suggestion(non-blocking):** update the default

You changed the default to `10` so we should update the README too

```suggestion
| [`monitoringPrecision`](./src/types.ts#L25) | `int` | millisecond | `10` | The approximate number of milliseconds for which to calculate event loop utilization averages. A larger value will result in more accurate averages at the expense of less granular data. Should be set to below the scrape interval of your metrics collector to avoid duplicated data points. |
```

Reply:
Fixed in 6b5fc8b290a27797fd91325929dffac664a92b75